### ISSUE TITLE
Update check function is broken on Java 15+

### ISSUE THREAD

**Issue by whalehub** [Querz/mcaselector#151]
### Describe the bug
When using MCA Selector with Java 15+, pressing the "check" button in the "About" dialogue window results in "Error" being displayed instead of "Up to date".

### Steps to reproduce the behavior
1. Install Java 15 from https://adoptopenjdk.net.
2. Install OpenJFX 11 from https://openjfx.io.
3. Install MCA Selector 1.13.2 from this repository.
4. Launch MCA Selector as follows from CMD:
```
"<path-to-java.exe>" --module-path "<path-to-openjfx-folder>\lib" --add-modules ALL-MODULE-PATH -jar "<path-to-mcaselector.jar>"
```

5. Click on "About".
6. Press on the "check" button.

### Expected behavior
After clicking the "check" button, "Up to date" should be displayed instead of "Error".

### Screenshots and other files
**Java 11**
![Java 11](https://xbb.datahoarder.dev/VUSu3/mitaCIGo92.png/raw)

**Java 15**
![Java 15](https://xbb.datahoarder.dev/VUSu3/zEXiziKi96.png/raw)

### Environment
- **OS:** Windows 10
- **Java version:** 15
- **Version:** 1.13.2

### Additional context
After clicking the "check" button, the following message is shown in the console when using Java 11
```
Warning: Nashorn engine is planned to be removed from a future JDK release
```

and the following error is shown in the console when using Java 15+
```java
java.lang.Exception: failed to check for latest version
        at net.querz.mcaselector.debug.Debug.dumpException(Debug.java:90)
        at net.querz.mcaselector.ui.dialog.AboutDialog.lambda$handleCheckUpdate$5(AboutDialog.java:105)
        at net.querz.mcaselector.ui.dialog.AboutDialog$$Lambda$507/00000000E9A027F0.run(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source)
java.lang.NullPointerException
        at net.querz.mcaselector.github.VersionChecker.parseJson(VersionChecker.java:48)
        at net.querz.mcaselector.github.VersionChecker.fetchLatestVersion(VersionChecker.java:42)
        at net.querz.mcaselector.ui.dialog.AboutDialog.lambda$handleCheckUpdate$5(AboutDialog.java:87)
        at net.querz.mcaselector.ui.dialog.AboutDialog$$Lambda$507/00000000E9A027F0.run(Unknown Source)
        at java.base/java.lang.Thread.run(Unknown Source)
```

The Nashorn engine was first [deprecated](https://openjdk.java.net/jeps/335) in Java 11 and then [removed](https://openjdk.java.net/jeps/372) in Java 15.

**Comment by bc-185** [comment]
Same when using the JRE bundled with Minecraft.

**Comment by Querz** [comment]
I honestly tried to avoid using a 3rd party library just to decode a bit of json, but it seems there is no way around that now.

**Comment by Querz** [comment]
Fixed in [1.13.3](https://github.com/Querz/mcaselector/releases/tag/1.13.3).
